radis/lbl/loader.py

```diff
@@ -1287,6 +1288,9 @@ def _check_database_params(
         # Check input types are correct
         if isinstance(path, string_types):  # make it a list
             path = get_files_from_regex(path)
+            for i in range(len(path)):
```

Comment:
you shouldn't use getTestFile (or any test related function) in the main package. Users may have stored their database files somewhere else.

Reply:
Made changes like you suggested.